<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditPort">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditIP">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Message</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>connect to IP and port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>mystery button</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="serverclientChat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
